sktime/forecasting/tests/test_exp_smoothing.py

```diff
@@ -57,3 +57,34 @@ def check_panel_expsmooth():
     forecaster = ExponentialSmoothing(trend="add", sp=1)
     forecaster.fit(y)
     forecaster.predict(fh=[1, 3])
+
+
+@pytest.mark.skipif(
+    not _check_soft_dependencies("statsmodels", severity="none"),
+    reason="skip test if required soft dependency not available",
+)
+def check_panel_with_freq():
```

Comment:
if this is a failure case for the bug, could you move it to `test_base`? Optimally, it should also not have a soft dependency - given the nature of th bug, I suspect any estimator would trigger it, e.g., `NaiveForecaster`?

Reply:
added 👍  uncovered some issues when updating the forecaster in that case where the frequency is changed when combing new and old data, e.g. from 24H to 1D, which causes issues in predict() where the timestamps are internally changed to period. Fixed it tho I think

Reply:
can you add tests covering that directly?

Reply:
yeah added into `sktime\datatypes\tests\test_utils.py`